content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+
+Since then, Sig Storage and other Sigs are working extra hard to ensure the feature land in great spots. A lot of effort has been put in by the maintainers of each plugin towards making the CSI Migration feature on by default & generally available for their plugins. This blog is intended to give a status update to the feature as well as changes since Kubernetes 1.17 to 1.23. In addition, it will also cover the future roadmap for the CSI migration feature GA for each storage plugins.
+
+## Quick recap: What is CSI Migration and Why?
+
+[Container Storage Interface](/blog/2019/01/15/container-storage-interface-ga/) has been GA since Kubernetes v1.13. CSI volume plugin was introduced to make Kubernetes volume plugins more maintainable and truly extensible. It is designed to replace the in-tree storage plugin especially vendor specific plugins for better maintainability(independent release cycle and support) as well as less vulnerability(security/reliability bugs can exist).
+
+As more CSI Drivers were created and became production ready, we wanted all Kubernetes users to benefit from the CSI model. However, we did not want to force users into making workload/configuration changes by breaking the existing generally available storage APIs. The way forward was clear - we would have to replace the backend of the “in-tree plugin” APIs with CSI and make the existing storage API/Interface working as usual.
+
+The CSI migration effort enables the replacement of existing in-tree storage plugins such as `kubernetes.io/gce-pd` or `kubernetes.io/aws-ebs` with a corresponding [CSI driver](https://kubernetes-csi.github.io/docs/introduction.html) from the storage backend. If CSI Migration is working properly, Kubernetes end users shouldn’t notice a difference. When a Kubernetes cluster administrator updates a cluster to enable CSI migration, existing stateful deployments and workloads continue to function as they always have; however, behind the scenes Kubernetes hands control of all storage management operations (previously targeting in-tree drivers) to CSI drivers.
+
+For example, Suppose you are a `kubernetes.io/gce-pd` user, after CSI migration, you can still use `kubernetes.io/gce-pd` to provision new volumes, mount existing GCE-PD volumes or delete existing volumes. All existing API/Interface will still function correctly. However, the underlying function calls are all going through the [GCE PD CSI driver](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) instead of the in-tree Kubernetes function.
+
+This enables a smooth transition for end users. Additionally as storage plugin developers, we can reduce the burden of maintaining the in-tree storage plugins and eventually remove them from the core Kubernetes binary.
+
+## What has been changed and What's new?
+
+Since Kubernetes v1.17, there are some new changes introduced:
+
+### New Feature Gate
```

Comment:
While this section does a good job explaining both gates, IMO it'd be even better if we reorganize it to jump straight to the point in the first paragraph. Perhaps moving the last paragraph up here would help.